user_data/hyperopts/sample_hyperopt.py

```diff
@@ -1,17 +1,31 @@
 # pragma pylint: disable=missing-docstring, invalid-name, pointless-string-statement
 
-import talib.abstract as ta
-from pandas import DataFrame
-from typing import Dict, Any, Callable, List
 from functools import reduce
+from math import exp
+from typing import Any, Callable, Dict, List
+from datetime import datetime
 
-import numpy
+import numpy as np# noqa F401
+import talib.abstract as ta
+from pandas import DataFrame
 from skopt.space import Categorical, Dimension, Integer, Real
 
 import freqtrade.vendor.qtpylib.indicators as qtpylib
 from freqtrade.optimize.hyperopt_interface import IHyperOpt
 
-class_name = 'SampleHyperOpts'
+# set TARGET_TRADES to suit your number concurrent trades so its realistic
+# to the number of days
+TARGET_TRADES = 600
+# This is assumed to be expected avg profit * expected trade count.
+# For example, for 0.35% avg per trade (or 0.0035 as ratio) and 1100 trades,
+# self.expected_max_profit = 3.85
+# Check that the reported Σ% values do not exceed this!
+# Note, this is ratio. 3.85 stated above means 385Σ%.
+EXPECTED_MAX_PROFIT = 3.0
+
+# max average trade duration in minutes
+# if eval ends with higher value, we consider it a failed eval
+MAX_ACCEPTED_TRADE_DURATION = 300
```

Comment:
A minor addtional note: do we need these constants in the hyperopt? It's not the loss...

Reply:
no ... forgot to remove them after the refactoring, good catch!